.github/workflows/publish.yml

```diff
@@ -49,17 +47,9 @@ jobs:
           script: |
             const version = process.env.tagName.replace(/^v/i, '');
             core.setOutput('version', version)
-      - name: Setup
-        uses: ./.github/actions/setup
-        with:
-          enable-build-cache: 'true'
-          install-jdk: 'true'
-      - name: Build Swagger CodeGen
-        run: yarn build:swagger-gen
-      - name: Build
-        run: yarn build
-      - name: Set new version
-        run: yarn set:version ${{ steps.get-version.outputs.version }}
+      - uses: ./.github/actions/setup
+      - uses: ./.github/actions/download-build-output
+      - run: yarn set:version ${{ steps.get-version.outputs.version }}
```

Comment:
you can change the `is-prerelease` (in following lines) to `is-prerelease: ${{ inputs.prerelease }}`

Reply:
done